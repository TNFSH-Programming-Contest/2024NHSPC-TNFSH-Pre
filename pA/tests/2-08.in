839027665155459109 480631763770540121 843217004946831270
113870170165512091 326318143008842321 272373130718582052
