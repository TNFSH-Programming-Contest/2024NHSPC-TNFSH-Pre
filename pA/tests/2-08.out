712561443892936464
